assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
I reverted this in 3412d7b as it was messing up the sidebar navigation. I think the original intention of lists being unstyled by default is sound, as sometimes you'll want a list to have semantic structure but not styling. 

Maybe we'll have to document using a bit of HTML5. Can you think of another way to do this? Use something besides an article? Use a class for styled or unstyled?